ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -122,6 +122,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-role
+- as: e2e-aws-ipi-disasterrecovery-p2
+  cron: 0 1 * * 2
```

Comment:
`cron: 0 1 */7 * *`

Reply:
I want the job is executed on Tuesday weekly,.

Reply:
I suggest change it to cron: "0 1 * * 2,5", it may cover errata nightly build test.

Reply:
ok. so, it should be `cron: 0 1 * * 2`.
`0 1 * * 2,5` means twice every week, but P2 is one every week.

Reply:
Yes, new rule for 4.12, https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private